public/javascripts/controllers/buyer-ctrl.js

```diff
@@ -109,11 +110,11 @@ angular.module('jasmic.controllers')
      *  TODO: Document this controller!
      */
     .controller('BuyerProfileCtrl', ['$location','$scope', '$mdDialog','$routeParams', 'BuyerFactory',
-        'BuyerTypesListingFactory', 'OpenTransactionsFactory', 'TransactionsFactory', 'RepFactory', 'CropsFactory', 'UnitsFactory',
-        'BuyerDemandFactory', 'DemandsFactory',
+        'BuyerTypesListingFactory', 'OpenTransactionsFactory', 'TransactionsFactory', 'RepFactory', 'RepEditFactory', 'CropsFactory', 'UnitsFactory',
+        'BuyerDemandFactory', 'DemandEditFactory', 'DemandsFactory',
```

Comment:
@nick22891 A suggestion here, instead of write more services/factories (DemandEditFactory), we could add the update verb to the DemandFactory already in place, since the DemandFactory shows a Demand by id. Less lines of code = Less bugs.